src/style.css

```diff
@@ -1,73 +1,126 @@
+/* ------------ youtube.com/CodeExplained ------------ */
 body {
-  background-color: rgb(241, 233, 233);
+  padding: 0;
+  margin: 0;
+  background-color: rgba(0, 0, 0, 0.1);
```

Comment:
- [ ] Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. ❌

If I open the local store it shows me an index of 0 for all your tasks. If I add it also needs to increase:

> add
> task 1 = index 1
> task 2 = index 2 ...

> delete tasks 2 
> remaining tasks index = 1

If I delete the item from the list. It also refreshes the indexes of the tasks.